### DISCUSSION TITLE
Looking for some scrips (using in MTEX5.4) of deformed rock minerals (such as  quartz, calcite, muscovite and feldspar) for plotting pole figures

### DISCUSSION THREAD

**Discussion by Caoxw2020** [mtex-toolbox/mtex#797] [Ask Anything]
Hello everyone:
      I am a PhD students and major is structural geology. I am doing some research in Himalaya, and looking for some scrips (using in MTEX5.4) of deformed rock minerals (such as  quartz, calcite, muscovite and feldspar) for plotting pole figures, Anyone who under take the similar research and share with me ?  Thanks a lot.

**Comment by lfgmorales** [comment]
Dear Caoxw2020


Although you may find people willing to share their scripts with you, how can you be sure that they do the calculations correctly? How would you know that person A or B defined whatever parameters in the script correctly? There are a lot of background information particular for your samples that have to be defined correctly from the beginning, and applying someone else's script to your samples might be a risky business, ending up with results that you won't be able to explain


A better approach to learn MTEX is to follow the extensive documentation, and when you face a problem, you come to the forum and ask your questions, this way you can be sure your calculations will be correct, and if someone ask you details about your calculations, you will be able to explain.


the documentation can be found here


https://mtex-toolbox.github.io/Documentation.html


Best



Luiz




Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________

**Reply by Caoxw2020** [reply]
Yes, Thank you. I can plot the pole figure of quartz. But now I want to calculate the calcite, so I change the code, but can't run well, can you help me check out ? I have tried many times, always hints me CS_Calcite=ebsd326('Calcite-new').CS;

% Get CS and ODF for Calcite
CS_Calcite=ebsd326('Calcite-new').CS;
odf326_Calcite=calcODF(ebsd326('Calcite-new').orientations,'halfwidth',10*degree);
% Poles to planes can be specified by the option 'pole' or 'hkl' 
PF1Cal_c = Miller(0,0,0,1,CS_Calcite,'uvw');
PF2Cal_a = Miller(1,1,2,0,CS_Calcite,'hkl');
PF3Cal_r = Miller(1,0,-1,4,CS_Calcite,'hkl');
PF4Cal_e = Miller(0,1,-1,8,CS_Calcite,'hkl');
PFs_Calcite = {PF1Cal_c,PF2Cal_a,PF3Cal_r,PF4Cal_e};
% Plot figures colorful
plotPDF(odf326_Calcite,PFs_Calcite,'lower','resolution',2*degree,'contourf');
CLim(gcm,'equal');
mtexColorbar

**Comment by lfgmorales** [comment]
Are you sure the phase is called „Calcite-new“???

That looks like you onl change the mineral name, quartz from Oxford system is named „quartz-new“, but I am almost sure calcite is called calcite, and not calcite-new

Check the head of your script for the phase names


Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 20 Feb 2021, at 16:34, Caoxw2020 <notifications@github.com> wrote:

﻿

Yes, Thank you. I can plot the pole figure of quartz. But now I want to calculate the calcite, so I change the code, but can't run well, can you help me check out ? I have tried many times, always hints me CS_Calcite=ebsd326('Calcite-new').CS;

% Get CS and ODF for Calcite
CS_Calcite=ebsd326('Calcite-new').CS;
odf326_Calcite=calcODF(ebsd326('Calcite-new').orientations,'halfwidth',10degree);
% Poles to planes can be specified by the option 'pole' or 'hkl'
PF1Cal_c = Miller(0,0,0,1,CS_Calcite,'uvw');
PF2Cal_a = Miller(1,1,2,0,CS_Calcite,'hkl');
PF3Cal_r = Miller(1,0,-1,4,CS_Calcite,'hkl');
PF4Cal_e = Miller(0,1,-1,8,CS_Calcite,'hkl');
PFs_Calcite = {PF1Cal_c,PF2Cal_a,PF3Cal_r,PF4Cal_e};
% Plot figures colorful
plotPDF(odf326_Calcite,PFs_Calcite,'lower','resolution',2degree,'contourf');
CLim(gcm,'equal');
mtexColorbar

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/797#discussioncomment-388553>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN7AEDA3NYT3RV2SE43S77I7XANCNFSM4X56AFCQ>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/797#discussioncomment-388553", "url": "https://github.com/mtex-toolbox/mtex/discussions/797#discussioncomment-388553", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
